.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Wonderful you started using prettier! Hope it works well with JS on your setup. :)

Reply:
I use Beautify :) Prettier does't work in my VSC.

Reply:
Oh! I see. Maybe when we meet next time I will be able to help? What problems do you have exactly? Anyways, if you don't feel you need prettier, it's absolutely fine, it's just supposed to help, not be an additional trouble-maker. :)